openapi/components/examples/search/result.yaml

```diff
@@ -0,0 +1,208 @@
+value:
+  data:
+    - id: 1
+      type: badge
+      date: 2022-01-30T21:00:00Z
+      # TODO add a proper homepage for a badge
+      url: https://{hostname}.{domain}/badge/1/update
+      title: A badge
+      image: https://{hostname}.{domain}/static/images/badges/ui/25.svg
+      description: |
+        A badge in this Schoolbox instance.
+      creator:
+        id: 1
+        firstName: Schoolbox
+        lastName: Admin
+        fullName: Schoolbox Admin
+      context: []
+      icon: badge
+      # TODO implement API for badges
+      _links: {}
+    - id: 1
+      type: event
+      date: 2022-01-30T21:00:00Z
+      url: https://{hostname}.{domain}/calendar/event/1
+      title: An event
+      image: ~
+      description: |
+        An event in this Schoolbox instance.
+      creator:
+        id: 2
+        title: Ms
+        firstName: Teacher
+        lastName: User
+        fullName: Ms Teacher User
+      context: []
+      icon: calendar
+      _links: {}
+    - id: 1
+      type: news
+      date: 2022-01-30T21:00:00Z
+      url: https://{hostname}.{domain}/news/1
+      title: A news article
+      image: https://{hostname}.{domain}/storage/image.php?hash=1234567890&size=coverPhoto
+      description: |
+        A news article in this Schoolbox instance.
+      creator:
+        id: 2
+        title: Ms
+        firstName: Teacher
+        lastName: User
+        fullName: Ms Teacher User
+      context: []
+      icon: news
+      _links: {}
+    - id: 10
+      type: page.class
+      date: 2022-01-30T21:00:00Z
+      url: https://{hostname}.{domain}/homepage/10
+      title: A class page (CLASSCODE)
+      image: https://{hostname}.{domain}/storage/image.php?hash=1234567890&size=coverPhoto
+      description: ~
+      creator:
+        id: 2
+        title: Ms
+        firstName: Teacher
+        lastName: User
+        fullName: Ms Teacher User
+      context:
+        - id: 2
+          name: Classes
+          url: https://{hostname}.{domain}/homepage/2
+        - id: 3
+          name: Year 1
+          url: https://{hostname}.{domain}/homepage/3
+      icon: homepage
+      _links:
+        subResults: https://{hostname}.{domain}/api/search/folder/10
+      _subResults:
+        data:
+          - id: 1
+            type: assessment.dueWork
```

Comment:
This is definitely an _Assessment_ and not a _Learning Activity_?

Reply:
I am not sure :crying_cat_face:

It ought to be able to be changed fairly easily?